src/structures.rs

```diff
@@ -33,47 +38,60 @@ impl Timetable {
             .parse::<NaiveDate>()
             .expect("Could not parse start date");
 
+        let stops: Vec<_> = gtfs.stops
+            .iter()
+            .map(|stop| {
```

Comment:
even if it will be a bit more verbose, I think it's better to implement the `From` trait